templates/manifest.json

```diff
@@ -6,6 +6,9 @@
             "label": "Start with Python",
             "category": "python",
             "description": "Project template using the Apify SDK for Python",
+            "messages": {
+                "postCreate": "To install additional Python packages, you need to activate the virtual environment in the \".venv\" folder in the actor directory."
```

Comment:
What about adding a command on how to do it or a link with instruction or do you think the most developer know it?

Reply:
Well the problem is it's platform-specific, it's different on Windows and on Linux, so I wanted to avoid some complicated logic and let the user do it the way they know.

Reply:
Ideally we would show them the correct instructions for their OS and their shell, but I think that would be quite complex to do, maybe we can do it later.